src/olympia/shelves/serializers.py

```diff
@@ -0,0 +1,18 @@
+from rest_framework import serializers
+
+from olympia.shelves.models import Shelf, ShelfManagement
+
+
+class ShelfSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Shelf
+        fields = ['id', 'title', 'endpoint', 'criteria',
+                  'footer_text', 'footer_pathname']
+
+
+class HomepageSerializer(serializers.ModelSerializer):
```

Comment:
So, I don't think we actually need this Serializer at all - we won't be returning any data from it directly.  The `ShelfSerializer` has everything we need.